src/routes/settings.tsx

```
@@ -96,6 +103,22 @@ function SettingsPage() {
                         onChange={(e) => setIp(e.target.value)}
                     />
+                    {availableIps.length > 0 && (
+                        <div className="pt-2">
+                           <span className="text-xs opacity-50 block mb-1">Detected LAN IPs:</span>
+                           <div className="flex flex-wrap gap-2">
+                               {availableIps.map(addr => (
+                                   <button 
+                                       key={addr} 
+                                       className="btn btn-xs btn-outline"
+                                       onClick={() => setIp(addr)}
+                                   >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)